quantmsrescore/annotator.py

```
@@ -277,38 +254,66 @@ def annotate(self) -> None:
         try:
             out_path = Path(filename)
-            OpenMSHelper.write_idxml_file(
-                filename=out_path,
-                protein_ids=self._idxml_reader.openms_proteins,
-                peptide_ids=self._idxml_reader.openms_peptides,
-            )
-            logger.info(f"Annotated idXML file written to {out_path}")
+            pq.write_table(self._idparquet_psm, psm_file)
+            logger.info(f"psms.parquet file written to {out_path}")
+        except Exception as e:
+            logger.error(f"Failed to write psms.parquet psm file: {str(e)}")
```

Ruff (0.15.13)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)